packages/typed-env-cli/src/commands/gen-diff-env-name-type.ts

```diff
@@ -0,0 +1,74 @@
+import { Command, Flags } from '@oclif/core';
+import { parseEnv } from '../lib/utils/util';
+import path from 'path';
+import fs from 'fs';
+import { ensureDirSync } from '../lib/utils/fs';
+
+export default class GenDiffEnvNameType extends Command {
+  static override description = 'Generate diff env name type definition';
+
+  static override examples = [
+    `$ typed-env diff-env-name -p "$(env)" -a "$(env)"`,
```

Comment:
`typed-env diff-env-name -p "$(prev_env)" -a "$(env)"`
let's add example for output as well